### ISSUE TITLE
[Website] Schema Explorer: Ensures <img> elements have alternate text or a role of none or presentation.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6424]
### Target Platforms

NodeJS

### SDK Version

https://adaptivecards.io/  9/9/2021

### Application Name

Website

### Problem Description

[ADO 35903425](https://microsoft.visualstudio.com/OS/_workitems/edit/35903425)

**Repro Steps:**
1. Open above URL with valid credential.
2. Turn on the Narrator using ctrl+win+enter key.
3. Navigate to 'Schema Explorer' link and using arrow keys press enter key to activate it.

**Actual Result:**
Alt is not defined for 'Decorative image' under 'Publish Adaptive Card Schema.'

**Expected Result:**
Alt should be defined for the decorative image under 'Publish Adaptive Card Schema.'

### Card JSON

```JSON
No card
```

**Comment by jwoo-msft** [comment]
put it in deprecated bucket since we are using 1.0 schema that doesn't support alternate text in our schema explorer.
we could update the card to use the later version of the schema, however.
